boot/env.ml

```diff
@@ -78,12 +103,10 @@ let guess_coqlib () =
     check_file_else
       ~dir:Coq_config.coqlibsuffix
       ~file:prelude
-      (fun () ->
-         if Sys.file_exists (Filename.concat Coq_config.coqlib prelude)
-         then Coq_config.coqlib
-         else fail ())
+      (fun () -> relocate Coq_config.coqlib)
 
-(* Build layout uses coqlib = coqcorelib *)
+(* Build layout uses coqlib = coqcorelib
+   XXX we should be using -boot in build layout so is that dead code? *)
```

Comment:
The whole stuff about coqcorelib and plugins should go away and instead ocamlfind be queried, I think, also I am not sure the code that adds the plugins_dir to `ocamlfind` should be kept.

Both changes shouldn't go here tho.

Reply:
I mean we are still passing `-I runtimelib` even if most of the time is already in `OCAMLPATH`